src/patternfly/components/Table/table.scss

```diff
@@ -715,20 +724,18 @@
 .#{$table} .#{$table}__draggable {
   --#{$table}--cell--MinWidth: 0;
   --#{$table}--cell--Width: 1%;
+
+  padding-inline-start: var(--#{$table}--cell--action--PaddingInlineStart);
+  padding-inline-end: var(--#{$table}--cell--action--PaddingInlineEnd);
 }
-// stylelint-enable
 
+// - Table favorite
 .#{$table} .#{$table}__favorite {
   --#{$table}--cell--MaxWidth: auto;
 }
 
 // - Table toggle
 .#{$table}__toggle {
-  --#{$table}--cell--PaddingBlockStart: var(--#{$table}__toggle--PaddingBlockStart);
-  --#{$table}--cell--PaddingBlockEnd: var(--#{$table}__toggle--PaddingBlockEnd);
-  --#{$table}--cell--PaddingInlineStart: var(--#{$table}__toggle--PaddingInlineStart);
-  --#{$table}--cell--PaddingInlineEnd: var(--#{$table}__toggle--PaddingInlineEnd);
-
```

Comment:
Toggle no longer needs a specific padding as it belongs to the `.#{$table}--cell--action` group.